### PR TITLE
Feature: Expose acceptedFiles and rejectedFiles to children

### PR DESCRIPTION
**Pull request by morulus**
**What kind of change does this PR introduce?**

- [ ] bugfix
- [x] feature
- [ ] refactoring / style
- [ ] build / chore
- [ ] documentation

**Did you add tests for your changes?**

- [x] Yes, my code is well tested
- [ ] Not relevant

**If relevant, did you update the documentation?**

- [ ] Yes, I've updated the documentation
- [x] Not relevant

**Summary**

Allow use accepted files and rejected files as props in children components.

```js
<Dropzone accept="image/*">
  {({ acceptedFiles }) => (acceptedFiles.length && <img src={acceptedFiles[0].preview} />)};
</Dropzone>
```

### CODE REVIEW

**Comment by okonet** [comment]
Can you please update the documentation? I think this change should be reflected in it.

**Comment by morulus** [comment]
I'm glad you found this useful.
> Can you please update the documentation?
Of course

**Comment by morulus** [comment]
[Done](https://github.com/okonet/react-dropzone/pull/380/commits/069af08fa9f501d1ba7d6e145df7572ad11d9581). But I do not understand why first string in readme marked as changed. Mystic.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-react-dropzone [![Build Status](https://travis-ci.org/okonet/react-dropzone.svg?branch=master)](https://travis-ci.org/okonet/react-dropzone) [![npm version](https://badge.fury.io/js/react-dropzone.svg)](https://badge.fury.io/js/react-dropzone) [![codecov](https://codecov.io/gh/okonet/react-dropzone/branch/master/graph/badge.svg)](https://codecov.io/gh/okonet/react-dropzone) [![OpenCollective](https://opencollective.com/react-dropzone/backers/badge.svg)](#backers) 
+react-dropzone [![Build Status](https://travis-ci.org/okonet/react-dropzone.svg?branch=master)](https://travis-ci.org/okonet/react-dropzone) [![npm version](https://badge.fury.io/js/react-dropzone.svg)](https://badge.fury.io/js/react-dropzone) [![codecov](https://codecov.io/gh/okonet/react-dropzone/branch/master/graph/badge.svg)](https://codecov.io/gh/okonet/react-dropzone) [![OpenCollective](https://opencollective.com/react-dropzone/backers/badge.svg)](#backers)
```
Comment (okonet) [inline README.md]: The white space gone missing. This is why it's marked as changed. Can you please put it back?

**Comment by okonet** [comment]
Can you please resolve rebase on latest master (and resolve conflicts) and put the space back in the README and it's good to merge! 👏

File: README.md (hunk 1 above)
Comment (morulus) [inline README.md]: Very strange, I did not delete this white space. I'll try to find it.

**Pull request merged**